src/models/participant/participantEndpoint.js

```diff
@@ -0,0 +1,81 @@
+/*****
```

Comment:
I also feel like these should be moved into a shared lib.

Reply:
I agree - let's cover it as a task in https://github.com/mojaloop/project/issues/848